testing/integrate.sh

```diff
@@ -628,16 +627,16 @@ fi
 mkdir -p ${LOGS}/exec-testdir
 ${SANSSH_PROXY} ${MULTI_TARGETS} --output-dir=${LOGS}/exec-testdir exec run /bin/sh -c "echo foo >&2"
 check_status $? /dev/null exec failed emitting to stderr
-if [ ! -f ${LOGS}/exec-testdir/0-localhost:50042 ] || [ ! -f ${LOGS}/exec-testdir/1-localhost:50042 ]; then
+if [ ! -f "${LOGS}/exec-testdir/0-localhost:50042;10s" ] || [ ! -f "${LOGS}/exec-testdir/1-localhost:50042;10s" ]; then
```

Comment:
I suspect creating files with a filename containing `;` will do more harm than good. Can we strip the duration out of the filename?

Reply:
Good point.